### ISSUE TITLE
Add support for writing out features in Excel 2007+

### ISSUE THREAD

**Issue by x97mdr** [x97mdr/pickles#45]
In order to distribute the features to business analysts who use Excel or who want to manipulate the tabular data
As a Pickles user
I want to generate the features inn Excel 2007+ format (.xlsx)

**Comment by dirkrombauts** [comment]
What's the status on this one?  This feature tops our company's "wish list" for pickles.

**Comment by x97mdr** [comment]
Hi Dirk,

I didn't realize anyone was waiting on it and I haven't had much time lately to see this through. It was next on my plate but I won't get a chance to work on it until Saturday or Sunday at the earliest.  Is that too late?

My sticking point was how to allow navigation and how to organize the features. I had initially thought of one feature per sheet with a navigation sheet but that didn't seem to jive well.  Any ideas on your end?  Once I figure out how to organize it then Developing it should be easy enough.

**Comment by dirkrombauts** [comment]
Hey, no problem about you not being aware of people wanting the excel thing - after all, if people don't tell you then how can you know?

About the organisation: I was thinking about one feature per sheet too.  TBH I hadn't really thought about navigation. The navigation sheet doesn't sound so bad too me.

BTW: I'd like to discuss a few programming-style issues of a philosophical nature with you, but perhaps the github issues board or the mailing list is not the best place for that.  Could you give me a shout at dirk dot rombauts at aimsoftware dot com?

**Comment by x97mdr** [comment]
After some feedback from Dirk it looks like we can call the Excel documentation output complete enough to merge into the trunk.  It needs a bit more work before releasing though : add more tests and allow tests results to be woven into the output

**Comment by x97mdr** [comment]
OK, all done for now.  might need some more polishing but this can be done as later pull requests.
